contracts/token/superfluid/SuperToken.sol

```diff
@@ -105,6 +135,12 @@ contract SuperToken is UUPSProxiable, SuperfluidToken, ISuperToken {
 		// register interfaces
 		ERC777Helper.register(address(this));
 
+		// deploy NFT proxies in SuperToken.initialize
+		// initialize the proxies, pointing to the canonical NFT logic contracts
+		// set in the constructor
+		// link the deployed NFT proxies to the SuperToken
+		_deployAndSetNFTProxyContractsIfUnset();
```

Comment:
this will not compile because `_deployAndSetNFTProxyContractsIfUnset` is not defined in this contract:
```
function _deployAndSetNFTProxyContractsIfUnset() internal {
    if (
        address(constantOutflowNFT) == address(0) &&
        address(constantInflowNFT) == address(0)
    ) {
        (
            address constantOutflowNFTProxyAddress,
            address constantInflowNFTProxyAddress
        ) = SuperfluidNFTDeployerLibrary
                .deployNFTProxyContractsAndInitialize(
                    SuperToken(address(this)),
                    address(CONSTANT_OUTFLOW_NFT_LOGIC),
                    address(CONSTANT_INFLOW_NFT_LOGIC)
                );
        constantOutflowNFT = IConstantOutflowNFT(
            constantOutflowNFTProxyAddress
        );
        constantInflowNFT = IConstantInflowNFT(
            constantInflowNFTProxyAddress
        );
    
        // emit NFT proxy creation events
        emit ConstantOutflowNFTCreated(constantOutflowNFT);
        emit ConstantInflowNFTCreated(constantInflowNFT);
    }
}
```

Reply:
deployment is now external